r/R/array.R

```diff
@@ -291,3 +299,11 @@ is.Array <- function(x, type = NULL) {
   }
   is_it
 }
+
+#' @export
+sort.Array <- function(x, decreasing = FALSE, na.last = TRUE, ...) {
+  if (!identical(na.last, TRUE)) {
+    stop("Arrow only supports sort() with na.last = TRUE", call. = FALSE)
```

Comment:
Do we need a C++ JIRA for this?

Is there an R workaround that would allow supporting this?

Reply:
I can work around it in R. I'll create a C++ JIRA too because controlling where missing values sort is common across different languages (e.g. `na_position` in pandas and `NULLS FIRST` and `NULLS LAST` in SQL)

Reply:
R workaround: 2ae2a96fc7ff5162de6cf3cf8f4cd7d9c45b44d0
Jira for the C++ library feature: https://issues.apache.org/jira/browse/ARROW-12063